components/pingdom/actions/create-http-check/create-http-check.mjs

```diff
@@ -0,0 +1,199 @@
+import {
+  PROBE_FILTERS_OPTIONS,
+  RESOLUTION_OPTIONS,
+} from "../../common/constants.mjs";
+import { removeHttp } from "../../common/utils.mjs";
+import pingdom from "../../pingdom.app.mjs";
+
+export default {
+  key: "pingdom-create-http-check",
+  name: "Create HTTP Check",
+  description: "Creates a new HTTP check in Pingdom. [See the documentation](https://docs.pingdom.com/api/#tag/Checks/operation/post.checks)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    pingdom,
+    host: {
+      type: "string",
+      label: "Host",
+      description: "The target host for the check.",
+    },
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The name of the check.",
+    },
+    auth: {
+      type: "string",
+      label: "Auth",
+      description: "Username and password, colon separated.",
+      optional: true,
+    },
+    customMessage: {
+      type: "string",
+      label: "Custom Message",
+      description: "Custom message that will be added to email and webhook alerts.",
+      optional: true,
+    },
+    integrationids: {
```

Comment:
We should use camelCase for the props where possible. There are several props here than can be updated to use camelCase.

```suggestion
    integrationIds: {
```